cfgov/v1/models/snippets.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+
+from wagtail.wagtailsnippets.models import register_snippet
+from wagtail.wagtailcore.fields import RichTextField, StreamField
+from wagtail.wagtailadmin.edit_handlers import FieldPanel, StreamFieldPanel
+
+from . import molecules
+
+
+@register_snippet
+class Contact(models.Model):
+    heading = models.CharField(max_length=22, blank=False)
```

Comment:
yea i think we should since a url has to be unique or you cannot route to it, but heading not so much.